public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx

```diff
@@ -34,37 +32,14 @@ const testCopyHasTemplate = (test: BannerTest, template: string): boolean =>
       (variant.mobileBannerContent && copyHasTemplate(variant.mobileBannerContent, template)),
   );
 
-interface BannerTestEditorProps {
-  test: BannerTest;
-  onChange: (updatedTest: BannerTest) => void;
-  onValidationChange: (isValid: boolean) => void;
-  visible: boolean;
-  editMode: boolean;
-  onDelete: () => void;
-  onArchive: () => void;
-  onTestSelected: (testName: string) => void;
-  testNames: string[];
-  testNicknames: string[];
-  createTest: (newTest: BannerTest) => void;
-}
-
-const BannerTestEditor: React.FC<BannerTestEditorProps> = ({
+const BannerTestEditor: React.FC<ValidatedTestEditorProps<BannerTest>> = ({
```

Comment:
Most of this file hasn't changed. It now takes the standard `ValidatedTestEditorProps`